src/main.ts

```diff
@@ -6,7 +6,7 @@ import * as path from 'path';
 
 import { createScriptFile, TEMP_DIRECTORY, NullOutstreamStringWritable, deleteFile, getCurrentTime } from './utils';
 
-const START_SCRIPT_EXECUTION_MARKER: string = 'Azure CLI GitHub Action: Starting script execution';
+const START_SCRIPT_EXECUTION_MARKER: string = `Starting script execution via docker image mcr.microsoft.com/azure-cli:`;

```

Comment:
why do you want to print this statement inside container. Better keep it the way it was. and add the suffix part only before running container

Reply:
echo inside the container is to draw a border between docker image download logs and user script logs. It is getting printed only once and not twice. If we remove it from within the container docker logs will also get printed